### DISCUSSION TITLE
How to find the Version

### DISCUSSION THREAD

**Discussion by EvanSlat** [coturn/coturn#1248] [Q&A]
In my application I was using the line "turnserver -h | grep Version" to pull what the coturn version was. yet after updating to Version Coturn-4.5.1.1 'dan Eider' that method stopped working and I am having problems finding a different way that works to get the coturn version. 

Is there any build-in method or way to find the coturn version?

**Comment by tyranron** [comment]
@EvanSlat we have some in [Docker tests](https://github.com/coturn/coturn/blob/docker/4.6.2-r4/docker/coturn/tests/main.bats#L44-L46):
> ```bash
> turnserver -o --log-file=stdout | grep -m 1 'Version Coturn' \
>                                 | cut -d ' ' -f6 \
>                                 | cut -d '-' -f2
> ```

But it could break with new versions once again, in the future.

**Reply by eakraly** [reply]
On "newer" versions you can `turnserver --version`
